CONTRIBUTING.md

```diff
@@ -1,32 +1,32 @@
 # Contributing
 
-When contributing to this repository, please read through and get familiarised with the processes of making a change to this package.
+Please read through and get familiarised with the processes for introducing a
+change in Castor.
 
-If you have further questions please discuss the change you wish to make with the owners of this repository.
+If you have further questions please discuss them with the owners of this
+repository.
 
 ## Get ready for development
 
-You will need Node 14.17 running locally.
+You will need the Node version specified in `.nvmrc` locally.
 
 **1. Install Node**
 
-Use [nvm](https://github.com/nvm-sh/nvm/) to manage the different Node versions and switch between them.
+We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage multiple Node
+versions.
 
-To install and switch to required version run:
+To install and switch to version specified in `.nvmrc` run:
 
     nvm install && nvm use
 
-Finally install Yarn via Homebrew:
+You'll also need [yarn](https://classic.yarnpkg.com/en/docs/install) which you
+can install with `npm`:
 
-    brew install yarn
-
-Then remove Node that was installed alongside Yarn:
-
-    brew uninstall node --ignore-dependencies
+    npm i -g yarn
```

Comment:
Isn't this for Yarn 2? Which does not work for us.

Reply:
No Yarn 2 wants us to use corepack https://yarnpkg.com/getting-started/install

Installing with `npm -g` is for v1 https://classic.yarnpkg.com/en/docs/install#mac-stable